src/olympia/discovery/migrations/0006_auto_20200807_2051.py

```diff
@@ -11,16 +11,20 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.CreateModel(
-            name='HomepageShelves',
-            fields=[
-            ],
-            options={
-                'verbose_name_plural': 'homepage shelves',
-                'proxy': True,
-                'indexes': [],
-                'constraints': [],
-            },
-            bases=('shelves.shelfmanagement',),
-        ),
+        # Have to disable this because it broke the migration chain when we deleted the
```

Comment:
mm does it make sense to keep this file then?

Reply:
If we delete the file we have to adjust any migrations that depend on it and if would cause problems locally if anyone has migrated up to that migration but not beyond.  So we could just delete all the code inside and have it an empty migration file.  Not sure if that's better than commenting it out.

Reply:
oh, mmm it is an old migration. Well, why would that fail now? We have 11 migrations for `discovery` and this `0006` seems to be at least 12 months old.

Reply:
Not sure, but, afaik, django parses the migrations each time and builds an internal data structure of what the models would be from the migration files, and then compares it to the reality defined in models.py.  In this case it can't resolve the base model for the proxy model defintion any longer so fails.  

It certainly seems like a django bug but some of the renaming I did when I replaced the proxy models for Shelf and ShelfManagement with a single proxy model might have fooled it somehow.

Proxy models are a bit weird anyway - I'm not really certain why they even need to be in migration files when they don't have any database representation

Reply:
Ok, I guess that's fine then